.pullapprove.yml

```diff
@@ -63,8 +63,8 @@ groups:
 
   ux-core:
     conditions:
-    - "'**/*.hbs' in files or 'static/styles/**/*.scss' in files or '.pullapprove.yml' in files"
-    # only review if "**/*.hbs", "static/styles/**/*.scss" files or ".pullapprove.yml" have been touched
+    - "'static/images/*/.' in files, theme/* or 'static/fonts/*/.' in files or 'static/styles//.scss' in files"
```

Comment:
- "'static/images/*/.' in files, 'theme/*' in files or 'static/fonts/**/*.ext' in files or 'static/styles//.scss' in files"